Sources/MapLibreSwiftDSL/Style Layers/Style Layer.swift

```diff
@@ -161,3 +161,13 @@ public extension StyleLayer {
         modified(self) { $0.insertionPosition = .belowOthers }
     }
 }
+
+public extension StyleLayerDefinition {
```

Comment:
Please review if this how you picture its usage, as I see other zoom level code in this spm already, but didn't find a way to access it from the MapView view builder.

Reply:
@Archdoog on our review call: "I think he is on to something fantastic" 😂  We definitely missed exposing this idiomatically in the high-level `StyleLayerDefinition`!

Capturing another random thought: it might make sense to split this file up since it's now got a bunch of random crap in it :) Especially for things like modifiers which might not be as obvious/discoverable. (Can do this later.)